### PR TITLE
Fix webhook validation error message

### PR DESCRIPTION
**Pull request by shellykatz**
Fixed the selfNodeRemediationConfig webhook validation to contain all fields with wrong values (instead of only the first one that was found wrong)

### CODE REVIEW

File: api/v1alpha1/selfnoderemediationconfig_webhook_test.go [hunk 1]
```diff
@@ -77,6 +94,33 @@ func testInvalidCRs(validationType string) {
...
+func testMultipleInvalidFields(validationType string) {
+	var errorMsg string
+	snrc := createDefaultSelfNodeRemediationConfigCR()
+
+	for _, item := range testItems2 {
+		item := item
+		setFieldValue(snrc, item.name, item.durationValue)
+		errorMsg += "\n" + item.name + " cannot be less than " + item.minDurationValue.String()
+	}
+	println(errorMsg)
```
Comment (razo7) [inline api/v1alpha1/selfnoderemediationconfig_webhook_test.go:106]: Is that necessary? I assume it was merely for debugging.

Comment (shellykatz) [reply api/v1alpha1/selfnoderemediationconfig_webhook_test.go:106]: good catch! thanks

**Pull request merged**